dockstore-webservice/src/main/java/io/openapi/api/impl/ServiceInfoApiServiceImpl.java

```diff
@@ -0,0 +1,78 @@
+package io.openapi.api.impl;
+
+import java.util.Date;
+import java.util.Optional;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+
+import io.dockstore.webservice.core.User;
+import io.openapi.api.ServiceInfoApiService;
+import io.openapi.model.Service;
+import io.openapi.model.ServiceType;
+
+public class ServiceInfoApiServiceImpl extends ServiceInfoApiService {
+    @Override
+    public Response getServiceInfo(SecurityContext securityContext, ContainerRequestContext value, Optional<User> user) {
+        Service service = getService();
+        return Response.ok().entity(service).build();
+    }
+
+    public static Service getService() {
+        Service service = new Service();
+        service.setId("1");
+        service.setName("Dockstore");
+        service.setType(getServiceType());
+        service.setDescription("The GA4GH Tool Registry API is a standard for listing and describing available tools (both stand-alone, Docker-based tools as well as workflows in CWL, WDL, Nextflow, or Galaxy) in a given registry,");
+        service.setOrganization(getOrganization());
+        service.setContactUrl("https://discuss.dockstore.org/t/opening-helpdesk-tickets/1506");
+        service.setDocumentationUrl("https://docs.dockstore.org/en/develop");
+        // TODO: Somehow get swagger codegen to generate string parameter (RFC 3339) instead of Date
+        service.setCreatedAt(new Date());
+        // TODO: Somehow get swagger codegen to generate string parameter (RFC 3339) instead of Date
+        service.setUpdatedAt(new Date());
```

Comment:
"Last updated" not sure if it means last time the implementation of the API changed or when content changed

Reply:
As in https://github.com/ga4gh-discovery/ga4gh-service-info/blob/develop/service-info.yaml#L93 seems ambiguous
Is it when the standard changed?
When the API implementation of the standard changed?
When the content backing the API changed?

Worth asking a question on the ga4gh-service-info repo via issue